### PR TITLE
WIP: Expand swagger file. Add script to call WSO2 API Manager endpoints

### PR DESCRIPTION
**Pull request by smcclellan7**

### CODE REVIEW

File: extras/wso2/api_client.sh [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+echo "Subscribe Response: $subscribe_resp"
+
+# Get auth token
+
+get_token_uri=$(get_uri "https" 8243 "/token")
+get_token_params="grant_type=password"
+get_token_params="$get_token_params&username=$username"
+get_token_params="$get_token_params&password=$password"
+if [ ! -z $scope ]; then
+  # TODO(smcclellan): This isn't working for some reason. The returned token always has default scope.
```
Comment (smcclellan7) [inline extras/wso2/api_client.sh]: I filed a ticket with WSO2 about this: https://wso2.org/jira/browse/APIMANAGER-5373

**Comment by mpkocher** [comment]
@smcclellan7  Do you think this is good enough for version1?

**Comment by smcclellan7** [comment]
@mpkocher I think so.

**Comment by mpkocher** [comment]
Great thanks.

**Pull request merged**